pkg/serverless/trace/propagation/extractor.go

```diff
@@ -112,6 +113,11 @@ func (e Extractor) extract(event interface{}) (*TraceContext, error) {
 		carrier, err = headersCarrier(ev.Headers)
 	case events.LambdaFunctionURLRequest:
 		carrier, err = headersCarrier(ev.Headers)
+	case events.StepFunctionEvent:
```

Comment:
I am not a Go expert. Does it mean that if the event include three StepFunctions specific metadata, it matches the StepFunctionEvent?

Reply:
I was mistaken -- that happens in serverless/trigger/events.go in https://github.com/DataDog/datadog-agent/pull/27988/files#diff-77b95cca617aa4d8daf59d6bf15540a3504815b769b963c5b78679dc1819fe21R277-R284